### ISSUE TITLE
Calculated GHG emission is inconsistent with calculated CO2 emission

### ISSUE THREAD

**Issue by Ceilican** [OiWorld/CarbonFootprint#117]
![screen shot 2016-10-11 at 17 44 22](https://cloud.githubusercontent.com/assets/1809436/19261069/7393cb08-8fda-11e6-8b13-bed5cebb7e5d.png)

**Comment by Ceilican** [comment]
GHG emissions should be always larger than CO2 emissions, because GHG includes CO2 and other gases.

**Comment by Ceilican** [comment]
@PrateekGupta1509 , could you have a look on this issue?

**Comment by PrateekGupta1509** [comment]
@Ceilican I will try to rectify this, I think there was inconsistency in the information on the [website](http://www.biomassenergycentre.org.uk/portal/page?_pageid=75,163182&_dad=portal&_schema=PORTAL) mentioned in issue #3. Also the server for the site is down, so I will look it into it if it up in next few days or go with some different data source to correct this.

**Comment by Ceilican** [comment]
Ok!

**Comment by PrateekGupta1509** [comment]
Hi @Ceilican! Sorry for the late communication, I had exams and my fest coming up so wasn't really active for a while.
The earlier website doesn't seem to host the information any longer. So I was thinking of using the same source we used for data released by the EPA [source-link](https://www.epa.gov/sites/production/files/2015-12/documents/emission-factors_nov_2015.pdf) and  shifting the GHG emission in terms of CO2e which is obtained by multiplying emission of each gas (CO2, CH4, N2O) by their respective global warming potential (GWP) and adding them up.
Also it might be better to shift the individual emission of CH4 and N2O in terms CO2e than in Kg/lbs as emission by these gases are better understood when compared with CO2.
The document above quotes: 

> Typically, greenhouse gas emissions are reported in units of carbon dioxide equivalent (CO2e). Gases are converted to CO2e by multiplying by their global warming potential (GWP). 

So I think it might be better to make this change as a more standard approach. Let me know your take on this so I can make the required change.

**Comment by Ceilican** [comment]
Hi @PrateekGupta1509 ! I agree with you. Feel free to proceed.

**Comment by PrateekGupta1509** [comment]
Hi @Ceilican I found an [archived version of the website](https://web.archive.org/web/20160616202256/http://www.biomassenergycentre.org.uk/portal/page?_pageid=75,163182&_dad=portal&_schema=PORTAL) we used as the source, and it clearly shows GHG emission in CO2e being less than the CO2 produced on combustion.

**Comment by Ceilican** [comment]
Hi @PrateekGupta1509 !  That table shows CO2 emissions *on combustion* and *life-cycle* GHG emissions (measured in CO2e).  *Life-cycle* emissions also take into account the emissions during production and transportation of the fuel, whereas *on combustion* emissions take only the emissions generated when the fuel is burned in the engine. For bio-fuels (like ethanol), the life-cycle emissions can be smaller than the emissions on combustion, because the production of bio-fuels absorbs CO2 from the atmosphere (e.g. in Brazil, ethanol is produced from sugar cane, and sugar cane absorbs CO2 when it grows). For fossil fuels (e.g. gasoline), the life-cycle emissions will always be greater than the emissions on combustion.

So:

1) there is still some inconsistency in our calculations, because the screenshot I included above shows CO2 emissions on combustion greater than life-cycle GHG emissions for *gasoline*, and this cannot be true.

2) We need to be more precise about whether we are talking about emissions on combustion or life-cycle emissions. (I have created issue #125 for that.)
